adminutils/adminutils.py

```diff
@@ -264,10 +283,66 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
         await ctx.tick()
 
     @emoji.command(name="remove")
-    async def emoji_remove(self, ctx, *, emoji: discord.Emoji):
+    async def emoji_remove(self, ctx: commands.Context, *, emoji: discord.Emoji):
         """Remove emoji from server"""
         if emoji.guild != ctx.guild:
             await ctx.send_help()
             return
         await emoji.delete(reason=get_audit_reason(ctx.author))
         await ctx.tick()
+
+    @commands.group()
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_channels=True)
+    @commands.bot_has_permissions(manage_channels=True)
+    async def channel(self, ctx: commands.Context):
+        """Manage channels"""
+        pass
+
+    @channel.command(name="create", aliases=["add"])
```

Comment:
This allows user to create a channel in category, which user doesnt have access to.

For example, like category "archive" on screenshot:
![yes](https://moeka.is-a-good-waifu.com/AQnELRD.png)